bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -311,8 +311,11 @@ spec:
                   return
             fi
             echo "Checking ${db} health"
-            serverid=$(ovsdb-tool show-log "${db}" | sed -ne '/server_id:/s/server_id: *\([[:xdigit:]]\+\)/\1/p')
-            match=$(ovsdb-tool show-log "${db}" | grep "prev_servers: *${serverid}(" || true)
+            # remove leading and trailing space around server_id
+            serverid=$(ovsdb-tool show-log "${db}" | sed -ne '/server_id:/s/ *server_id: *\([[:xdigit:]]\+\).*/\1/p')
+            # prev_servers is a list we are searching for " $server_id(" in the list
+            # prev_servers: 33b4("ssl:10.19.138.37:9643"), 720d("ssl:10.19.138.33:9643"), 7e4f("ssl:10.19.138.32:9643"
```

Comment:
ok, now I'm totally just nitpicking but presumably there's a "`)`" missing at the end of that line...

Reply:
This was copied from the original bugzilla.  I have yet to see multiple `prev_servers` in testing, but I checked the source and there should always be a closing paren.